Dockerfile

```diff
@@ -221,9 +209,9 @@ RUN apt update \
         php8.2-xsl \
         php8.2-zip \
     # Set default PHP version
-    && update-alternatives --set php /usr/bin/php8.0 \
-    && update-alternatives --set phpize /usr/bin/phpize8.0 \
-    && update-alternatives --set php-config /usr/bin/php-config8.0 \
+    && update-alternatives --set php /usr/bin/php8.1 \
```

Comment:
Had to bump default PHP version as `roave/backward-compatibility-check` does not support PHP 8.0 anymore.